experimental/split_mlir/src/iree/split_mlir/Passes.td

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_SPLIT_MLIR_TRANSFORM_PASSES
+#define IREE_SPLIT_MLIR_TRANSFORM_PASSES
+
+include "mlir/Pass/PassBase.td"
+
+def OutlineFunctions :
+    Pass<"iree-outline-functions", "mlir::ModuleOp"> {
+  let summary = "Outline operations in separate function(s).";
```

Comment:
Add a description here explaining expected usage in conjunction with the bisect marking pass and exactly how the resulting outlined functions are wired together.